python_modules/libraries/dagster-pandas/dagster_pandas/data_frame.py

```diff
@@ -120,6 +125,30 @@ def create_dagster_pandas_dataframe_description(description, columns):
     return buildme
 
 
+def create_dagster_pandas_dataframe_metadata(
+    pandas_df: pd.DataFrame,
+) -> Union[TypeCheck, MetadataValue]:
+    """
+    This function takes a pandas DataFrame and returns its metadata as a Dagster TableSchema.
+
+    Args:
+        pandas_df (pandas.DataFrame): A pandas DataFrame for which to create metadata.
+
+    Returns:
+        Union[TypeCheck, MetadataValue]: If the input is not a pandas DataFrame, it returns a TypeCheck object
```

Comment:
I see that you updated the code to remove the `TypeCheck` case, but it needs to be removed from the docstring as well.